e2e/tests/caseManagementOrder_test.js

```diff
@@ -2,26 +2,108 @@ const config = require('../config.js');
+  reviewAgreedCaseManagementOrderEventPage.enterChangesRequested(changeRequestReason);
+  await I.completeEvent('Save and continue', {summary: 'Summary', description: 'Description'});
+  I.seeEventSubmissionConfirmation(config.applicationActions.reviewAgreedCmo);
+  caseViewPage.selectTab(caseViewPage.tabs.draftOrders);
+  assertDraftCMO(I, '1', '1 January 2020', returnedStatus);
+});
+
+Scenario('Local authority makes changes requested by the judge', async (I, caseViewPage, uploadCaseManagementOrderEventPage) => {
+  await I.navigateToCaseDetailsAs(config.swanseaLocalAuthorityUserOne, caseId);
+  caseViewPage.selectTab(caseViewPage.tabs.draftOrders);
+  assertDraftCMO(I, '1', '1 January 2020', returnedStatus);
+  await localAuthoritySendsAgreedCmo(I, caseViewPage, uploadCaseManagementOrderEventPage);
   I.seeEventSubmissionConfirmation(config.applicationActions.uploadCMO);
   caseViewPage.selectTab(caseViewPage.tabs.draftOrders);
-  I.seeInTab(['Draft Case Management Order 1', 'Order'], 'mockFile.docx');
-  I.seeInTab(['Draft Case Management Order 1', 'Hearing'], 'Case management hearing, 1 January 2020');
-  I.seeInTab(['Draft Case Management Order 1', 'Date sent'], dateFormat(new Date(), 'dd mmm yyyy'));
-  I.seeInTab(['Draft Case Management Order 1', 'Status'], 'With judge for approval');
-  I.seeInTab(['Draft Case Management Order 1', 'Judge'], 'Her Honour Judge Reed');
+  assertDraftCMO(I, '1', '1 January 2020', withJudgeStatus);
+  I.dontSeeElement(locate(`//p/a[text()="${linkLabel}"]`));
 });
+
+Scenario('Judge seals and sends the agreed CMO to parties', async (I, caseViewPage, uploadCaseManagementOrderEventPage, reviewAgreedCaseManagementOrderEventPage) => {
+  await I.navigateToCaseDetailsAs(config.judicaryUser, caseId);
+  await caseViewPage.goToNewActions(config.applicationActions.reviewAgreedCmo);
+  reviewAgreedCaseManagementOrderEventPage.selectSealCmo();
+  await I.completeEvent('Save and continue', {summary: 'Summary', description: 'Description'});
+  I.seeEventSubmissionConfirmation(config.applicationActions.reviewAgreedCmo);
+  caseViewPage.selectTab(caseViewPage.tabs.orders);
+  assertSealedCMO(I, '1', '1 March 2020');
+  assertSealedCMO(I, '2', '1 January 2020');
+});
+
+const localAuthoritySendsAgreedCmo = async function (I, caseViewPage, uploadCaseManagementOrderEventPage, hearingDate, multiHearings) {
+  await caseViewPage.goToNewActions(config.applicationActions.uploadCMO);
+
+  if (multiHearings) {
+    await uploadCaseManagementOrderEventPage.associateHearing(hearingDate);
+    await I.retryUntilExists(() => I.click('Continue'), '#uploadedCaseManagementOrder');
+  }
+
+  await uploadCaseManagementOrderEventPage.uploadCaseManagementOrder(config.testNonEmptyWordFile);
+  await I.completeEvent('Submit');
+};
+
+const assertDraftCMO = function (I, collectionId, hearingDate, status) {
+  I.seeInTab([`Draft Case Management Order ${collectionId}`, 'Order'], 'mockFile.docx');
```

Comment:
minor, you could have local const 
const draftCMO = `Draft Case Management Order ${collectionId}`;
and then 
I.seeInTab([draftCMO, 'Order'], 'mockFile.docx');